lib/rubygems.rb

```diff
@@ -156,6 +156,13 @@ module Gem
     specifications/default
   ].freeze
 
+  ##
+  # The default value for SOURCE_DATE_EPOCH if not specified.
+  # We want a date after 1980-01-01, to prevent issues with Zip files.
+  # This particular timestamp is for 1980-01-02 00:00:00 GMT.
+
+  DEFAULT_SOURCE_DATE_EPOCH = 315_619_200
```

Comment:
Is this value correct in timezones with leap seconds too?

Reply:
Unix timestamps are specifically relative to 1970-01-01 00:00:00 UTC. Other time zones are never relevant.